src/test/regress/expected/gist_optimizer.out

```diff
@@ -220,6 +220,94 @@ select b from gist_tbl where b <@ box(point(5,5), point(6,6));
  (6,6),(6,6)
 (21 rows)
 
+-- Also test an index-only knn-search
+explain (costs off)
+select b from gist_tbl where b <@ box(point(5,5), point(6,6))
+order by b <-> point(5.2, 5.91);
+                            QUERY PLAN                             
+-------------------------------------------------------------------
+ Result
+   ->  Gather Motion 3:1  (slice1; segments: 3)
+         Merge Key: ((b <-> '(5.2,5.91)'::point))
+         ->  Sort
+               Sort Key: ((b <-> '(5.2,5.91)'::point))
+               ->  Index Scan using gist_tbl_box_index on gist_tbl
+                     Index Cond: (b <@ '(6,6),(5,5)'::box)
+                     Filter: (b <@ '(6,6),(5,5)'::box)
+ Optimizer: Pivotal Optimizer (GPORCA)
+(9 rows)
```

Comment:
I think we need `set optimizer_enable_sort = off;` to force orca to use index only scan too.

Reply:
I don't think so, the point of the tests that commit d04c8ed originally added was to make the Postgres planner choose index-only scan itself.